### DISCUSSION TITLE
Can't disable the Tooltip on the Checkboxes

### DISCUSSION THREAD

**Discussion by tobi110289** [KevinVandy/mantine-react-table#435] [Q&A]
### mantine-react-table version

2.0.0-beta.1

### react & react-dom versions

18.2.0

### Describe the bug and the steps to reproduce it

I am trying to disable the tooltips on the individual rows and the SelectAll Checkboxes, but I have found no way to do that successfully. 
If I give these Props, there is a little "empty" box rendered (see screenshot)
```
mantineSelectAllCheckboxProps: { title: "" },
mantineSelectCheckboxProps: { title: "" },
```
![Screenshot 2024-08-13 at 14 54 16](https://github.com/user-attachments/assets/6983a228-8e23-40ce-bc3e-73ef00e81720)

I am looking forward to a simple solution!
Thank you very much

### Minimal, Reproducible Example - (Optional, but Recommended)

/

### Do you intend to try to help solve this bug with your own PR?

No, because I do not have time to dig into it

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by alessandrojcm** [comment]
@tobi110289 hey can you try with the latest beta and confirm if this still happens?

**Comment by tobi110289** [comment]
@alessandrojcm I updated to "mantine-react-table": "2.0.0-beta.6".
It unfortunately still has the same behavior with `mantineSelectCheckboxProps: { title: "" }`. :
<img width="62" alt="Screenshot 2024-08-30 at 21 38 46" src="https://github.com/user-attachments/assets/7d3d5d35-6db9-437a-842d-e158a8e8a493">

Is there a different syntax I could try?

**Comment by alessandrojcm** [comment]
> @alessandrojcm I updated to "mantine-react-table": "2.0.0-beta.6".
> It unfortunately still has the same behavior with `mantineSelectCheckboxProps: { title: "" }`. :
> <img width="62" alt="Screenshot 2024-08-30 at 21 38 46" src="https://github.com/user-attachments/assets/7d3d5d35-6db9-437a-842d-e158a8e8a493">
> 
> Is there a different syntax I could try?

No that's look like a bug I'll check it out

**Comment by tobi110289** [comment]
@alessandrojcm thanks! Appreciate it :)

**Comment by alessandrojcm** [comment]
Hey, so after looking into this, I don't think it is a bug. the label gets passed to the tooltip as-is, and at the moment there is no way of hiding the tooltip. Moreover, I don't think hiding the tooltip if the string is empty would be a good idea. However, adding a prop to hide the tooltip would be trivial, please feel free to to add a PR if that is something you would like :) (You can start here: https://github.com/KevinVandy/mantine-react-table/blob/v2/packages/mantine-react-table/src/components/inputs/MRT_SelectCheckbox.tsx)
